legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java

```diff
@@ -314,9 +339,30 @@ public static DataSpaceAnalysisResult analyzeDataSpace(Root_meta_pure_metamodel_
             excResult.description = executionContext._description();
             excResult.mapping = HelperModelBuilder.getElementFullPath(executionContext._mapping(), pureModel.getExecutionSupport());
             excResult.defaultRuntime = HelperModelBuilder.getElementFullPath(executionContext._defaultRuntime(), pureModel.getExecutionSupport());
+            Optional<org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElement> packageableRuntime = pureModelContextData.getElements().stream().filter(e -> e.getPath().equals(excResult.defaultRuntime) && e instanceof PackageableRuntime).findFirst();
+            if (packageableRuntime.isPresent() && packageableRuntime.get() instanceof PackageableRuntime)
+            {
+                PackageableRuntime runtime = (PackageableRuntime) packageableRuntime.get();
+                if (runtime.runtimeValue.connections != null && !runtime.runtimeValue.connections.isEmpty() && !runtime.runtimeValue.connections.get(0).storeConnections.isEmpty())
+                {
+                    String storePath = runtime.runtimeValue.connections.get(0).store.path;
+                    Connection connection = runtime.runtimeValue.connections.get(0).storeConnections.get(0).connection;
+                    if (connection instanceof ConnectionPointer)
```

Comment:
shouldnt we handle embedded connections here  as well ?

Reply:
Currently only support what this feature has covered before 
https://github.com/finos/legend-studio/pull/3508/files#diff-0e9be2294b5afd6e1687f59f976f0640a5e79aa8c6d5db88df497bad86319084L98

Reply:
can we include it tho. no reason i see it shouldnt be covered.